### DISCUSSION TITLE
Looking for a developer to do custom Luckysheet work

### DISCUSSION THREAD

**Discussion by shib71** [dream-num/Luckysheet#1086] [General]
I'm doing some work on a project that needs a spreadsheet, but I ran into a couple of problems with Luckysheet. Would any of the developers here be open to doing paid work on them? We're happy for the work to be contributed back to the community, either as core features or examples. The initial work would be to update Luckysheet so that:
* multiple instances can run on one page, as embedded spreadsheets (and hopefully with an option to expand each one to full screen for easier editing)
* the various dependencies can be easily packaged and self-hosted (so that they can be used online or offline)
* the data can be exported to an Excel file (probably using SheetJS)
* setting up charting (and figuring out a way to export charts)

We will need someone who is comfortable using English. We're based in Sydney, and although this would be remote work we will need the dev to be able to join calls regularly. So an overlapping timezone would help too.

Are there any devs here that would be open to a project like this?

**Comment by Pavelkoen** [comment]
Let me know if you're still looking for someone

**Reply by shib71** [reply]
Thanks @Pavelkoen, we've ended up figuring out a direction ourselves.
